code/game/jobs/job/support_chaplain.dm

```diff
@@ -34,6 +34,8 @@
 	var/obj/item/weapon/storage/bible/B = new /obj/item/weapon/storage/bible(H)
 
 	spawn()
+		H.equip_to_slot_or_del(B, slot_l_hand)
```

Comment:
I'd put this at the veryyyy end, and also put it in the backpack, so it doesn't potentially interfere with other hand equipping things.

`H.equip_to_slot_or_del(B, slot_in_backpack)`